modules/GT_TestGameState.php

```diff
@@ -3,12 +3,28 @@
  *
  */
 
+use GT\Managers\CardsManager;
+
 class GT_TestGameState
 {
   public function __construct($game, $players)
   {
     $this->game = $game;
     $this->players = $players;
+    $this->cardsCount = [
```

Comment:
Can this be called `cardsInstanceIndex` instead? or `cardsInstIdx`? These numbers are indexes into the instance array within the card type. I think `cardCount` implies something else, like the count of each type of card?

Reply:
I don't like shortening in variables names but yes, cardsInstanceIndex makes sense